packages/react/src/useMarkprompt.tsx

```diff
@@ -61,6 +62,8 @@ export interface UseMarkpromptResult {
   updateActiveSearchResult: Dispatch<SetStateAction<string | undefined>>;
   /** Set a new value for the prompt */
   updatePrompt: Dispatch<SetStateAction<string>>;
+  /** Submit user feedback */
+  submitFeedback: (helpful: boolean) => Promise<void>;
```

Comment:
Should we pass `promptId` as a parameter here? We want to be able to vote on previous chat responses as well.